3p/messaging.js

```diff
@@ -25,7 +25,7 @@ export function nonSensitiveDataPostMessage(type, opt_object) {
   }
   const object = opt_object || {};
   object.type = type;
-  object.sentinel = window.context.amp3pSentinel;
+  object.sentinel = window.context.sentinel;
```

Comment:
could you make this naming change in a separate PR? In case this renaming break anything, we can easily revert a smaller PR.